Scoop/Controller/Onboarding/ProfilePictureViewController.swift

```diff
@@ -92,15 +95,19 @@ class ProfilePictureViewController: OnboardingViewController {
                 print("\(user.firstName) has been updated")
                 self.dismiss(animated: true)
                 self.containerDelegate?.dismiss(animated: true)
-                NetworkManager.shared.profileDelegate?.updateProfile()
+                NetworkManager.shared.profileDelegate?.updateUserProfile()
+                self.uploadButton.backgroundColor = .scoopDarkGreen
+                self.loadingSpinner.stopAnimating()
             case .failure(let error):
                 print("Failed to Authenticate: \(error.localizedDescription)")
```

Comment:
Remember that these error messages are for the developer. It would be helpful to provide the name of the file in which the error is occurring.